### DISCUSSION TITLE
Does Rocket-core or BOOM implement hypervisor extension?

### DISCUSSION THREAD

**Discussion by sammy17** [chipsalliance/rocket-chip#3245] [Q&A] [answered]
Could you please let me know whether Rocket or BOOM implements the hypervisor extension? 
If so, how can I enable it in the config?

**Comment by ZenithalHourlyRate** [comment, marked as answer]
Rocket implemented hypervisor extension. You could enable it with `WithHypervisor` in https://github.com/chipsalliance/rocket-chip/blob/276b792cae45a8797cc84ede105cee5aa5fd8903/src/main/scala/subsystem/Configs.scala#L332
